tests/test_nodestats_smoke.py

```diff
@@ -0,0 +1,59 @@
+"""Smoke test."""
+
+import importlib.resources as pkg_resources
+from pathlib import Path
+
+import h5py
+import yaml
+from test_io import dict_almost_equal
+
+import topostats
+from topostats.io import LoadScans, hdf5_to_dict, read_yaml
+from topostats.processing import (
+    process_scan,
+)
+
+BASE_DIR = Path.cwd()
+SMOKE = BASE_DIR / "tests/smoke"
+
+
+def test_smoke(regtest, tmp_path, load_scan_spm: LoadScans) -> None:
```

Comment:
`test_smoke()` doesn't tell the reader what is being tested, its not smoke that is being tested.

```suggestion
def test_regression_dnatracing_with_nodetsats(regtest, tmp_path, load_scan_spm: LoadScans) -> None:
```